### PR TITLE
Automatically add channels axis at back for init_from_channels_at_back

### PR DESCRIPTION
**Pull request by grigorisg9gr**
Small change in init_from_channels_at_back() in Image class. 

Currently if less dimensions are provided it does not raise any error or warning, e.g. :

Image.init_from_channels_at_back(np.random.rand(20, 40)).shape

### CODE REVIEW

**Comment by patricksnape** [comment]
The way the image constructor works now is that if you provide a 2D (no channels) image, we just put an 'empty' channel at the front and assume you meant a 2D 1-channel image. Perhaps we should just do the same here? If `ndims == 2` then add an empty axis to the back?

**Comment by nontas** [comment]
I agree with @patricksnape . We should do exactly what the `Image` constructor does, i.e.
1. Add an empty "channels" axis in case the provided `array` has `ndim == 2`.
2. Raise a `ValueError` in case the provided `array` has `ndim < 2`.

File: menpo/image/base.py [hunk 1]
```diff
@@ -206,11 +206,8 @@ def channels_to_front(pixels):
     """
     if not isinstance(pixels, np.ndarray):
         pixels = np.array(pixels)
-    # Channels to axis 0
-    if pixels.ndim == 3:
-        pixels = np.require(np.rollaxis(pixels, -1), dtype=pixels.dtype,
-                            requirements=['C'])
-    return pixels
+    return np.require(np.rollaxis(pixels, -1), dtype=pixels.dtype,
```
Comment (patricksnape) [inline menpo/image/base.py]: This was required because that check prevented multi-channel 2D+ conversion.

**Comment by patricksnape** [comment]
@jabooth Ignore menpobot - Windows conda-build is having [troubles](https://github.com/conda/conda-build/issues/1339). Review please?

**Pull request merged**